Makefile

```
@@ -310,11 +311,8 @@ ifneq (${DATAPREP_VERSION_OVERRIDE},)
 ${DATAPREP_VERSION_FILE}: FORCE
 	@printf '%s\n' "${DATAPREP_VERSION_OVERRIDE}" > ${DATAPREP_VERSION_FILE}
 else
-${DATAPREP_VERSION_FILE}: ${DATAPREP_PATH}/Makefile ${DATAPREP_PATH}/projects/deces-dataprep/recipes/deces_dataprep.yml ${DATAPREP_PATH}/projects/deces-dataprep/datasets/deces_index.yml
-	@cat ${DATAPREP_PATH}/Makefile\
-		${DATAPREP_PATH}/projects/deces-dataprep/recipes/deces_dataprep.yml\
-		${DATAPREP_PATH}/projects/deces-dataprep/datasets/deces_index.yml\
-	| sha1sum | awk '{print $1}' | cut -c-8 > ${DATAPREP_VERSION_FILE}
+${DATAPREP_VERSION_FILE}: $(addprefix ${DATAPREP_PATH}/,${dataprep_version_inputs})
```

checkmake (0.3.2)

[warning] Target "${DATAPREP_VERSION_FILE}" defined multiple times (lines 311 and 314).

(uniquetargets)